Resources/Private/JavaScript/Host/Containers/ContentCanvas/InlineUI/MarkActiveNodeAsFocused/index.js

```diff
@@ -0,0 +1,32 @@
+import {Component} from 'react';
+import {connect} from 'react-redux';
+import {CR} from 'Host/Selectors/index';
+import {$transform} from 'plow-js';
+import style from './style.css';
+import {node as NodePropType} from 'Shared/PropTypes/CR/Node/index';
+
+@connect($transform({
+    focusedNode: CR.Nodes.focusedSelector
+}))
+export default class MarkActiveNodeAsFocused extends Component {
```

Comment:
well, we need to listen to changes in the Store -- so if we did not use a React component, we'd need to manually hook into the store similar to what is done in https://github.com/reactjs/react-redux/blob/master/src/components/connect.js#L80 -- thus I thought it was better/easier to wrap it in a component.